src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+  background-color: #cecece;
+}
+
+header {
+  width: 100%;
+  box-sizing: border-box;
+  background-color: rgb(255, 255, 255);
+  max-height: 60px;
+  display: flex;
+  overflow: hidden;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  margin: 10px auto 10px 50px;
+}
+
+nav {
```

Comment:
Please, check the [checklist to the task](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) . Let`s use classes instead of tags for selectors . Fix it through all your file 
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/c2bb4df1-2818-4079-9f25-728aec3e1c47)